index.js

```diff
@@ -87,7 +87,7 @@ function dev (opts) {
 function log (ctx, start, len, err, event) {
   // get the status code of the response
   const status = err
-    ? (err.status || 500)
+    ? (err.isBoom ? err.output.statusCode : err.status || 500)
```

Comment:
@OmgImAlexis can you add a test please :)

Reply:
Should I add boom as a dep for the test?

Reply:
> Should I add boom as a dep for the test?

You can add `boom` as a devDep :)